### PR TITLE
{devel}[GCC/10.2.0] Jansson bumped up to version 2.13.1 

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/j/Jansson/Jansson-2.13.1-GCC-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * Full Unicode support (UTF-8)
+ * Extensive test suite"""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/akheron/jansson/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['f22901582138e3203959c9257cf83eba9929ac41d7be4a42557213a22ebcc7a0']
+
+preconfigopts = 'autoreconf -i && '
```
Comment (Micket) [inline easybuild/easyconfigs/j/Jansson/Jansson-2.13.1-GCC-10.2.0.eb:25]: Since this calls preconfigopts, this needs a to add builddep on Autotools

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/j/Jansson/Jansson-2.13.1-GCC-10.2.0.eb (hunk 1 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/j/Jansson/Jansson-2.13.1-GCC-10.2.0.eb:25]: Added Autoconf, thanks for spotting that.

Comment (sassy-crick) [reply easybuild/easyconfigs/j/Jansson/Jansson-2.13.1-GCC-10.2.0.eb:25]: Swapped Autoconf with Autotools as suggested.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/34d4b6422d043f2e1a0c7ede7bb34e56 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**